apps/plugin-browser/webpack.config.js

```diff
@@ -143,10 +143,16 @@ const options = {
     extensions: fileExtensions
       .map((extension) => `.${extension}`)
       .concat([".js", ".jsx", ".ts", ".tsx", ".css"]),
+    fallback: {
+      "process/browser": require.resolve("process/browser"),
+    },
   },
   plugins: [
     isDevelopment && new ReactRefreshWebpackPlugin(),
     new CleanWebpackPlugin({ verbose: false }),
+    new webpack.ProvidePlugin({
+      process: "process/browser",
+    }),
```

Comment:
These were required to prevent the imported packages from crashing which were accessing `process.env`.

Reply:
Ideally we would fix these in the packages if they are supposed to be isomorphic, because `process.env` doesn't exist in the browser. It just happens to in NextJS where we have used them so far. I think we're probably better off catching invalid process access in anything that's supposed to be browser-usable, otherwise we'll encounter this problem in any other browser app that doesn't happen to have `process.env` polyfilled or faked by something.